libraries/botbuilder/tests/teamsActivityHandler.test.js

```diff
@@ -508,6 +508,42 @@ describe('TeamsActivityHandler', () => {
                 .catch(err => done(err));
         });
 
+        it('handleTeamsTabFetch is not overridden', done => {

```

Comment:
Prefer `async` for tests (unless you _really_ can't get around not having `done`.

```typescript
await adapter.send().assertReply();
```

Should work. Don't feel like you have to rewrite this entire test file, I'd be fine with just updating the new tests.

Reply:
I saw that and agree on rewriting the new tests but draw a conditional line on updating the new tests. If the the amount of work is large I'd rather see another PR for the older tests.

Though we can always filter commits during the code review, so if you made the updates to the old tests on an isolated commit in the PR that works too.

Reply:
Yeah, I don't love that test file in general. I wasn't sure whether writing "right" tests was better than matching the existing ones. I made the `done => async` change but there's still a lot of issues I'd like to fix in there. I'll make an issue for it and do it in a separate PR.